tests/behat/commen_history_backup_restore.feature

```diff
@@ -12,7 +12,8 @@ Feature: Restore of studentquizzes in moodle exports with comment histories
   @javascript @_file_upload @_switch_window
   Scenario: Restore moodle backups containing history comments.
     Given I am on the "C1" "Course" page logged in as "admin"
-    And I navigate to "Restore" in current page administration
+    And I navigate to "Course reuse" in current page administration
+    And I select "Restore" from the "jump" singleselect
     And I press "Manage backup files"
     And I upload "mod/studentquiz/tests/fixtures/backup-moodle311-c1-historycomment.mbz" file to "Files" filemanager
```

Comment:
Rather than all these many steps of Behat code, can't we use one of the standard steps like

`And I restore "test_backup.mbz" backup into "Course 2" course using this options:`

I am not sure exactly which of the steps from backup/util/ui/tests/behat/behat_backup.php does what we need, but hopefully there is one.

Reply:
Also, please can we rename this file, so that 'comment' is spelt correctly ;-)

Reply:
Hi Tim, I will rename the files.
regarding the behat step. I had tried to find all the step in backup/util/ui/tests/behat/behat_backup.php
You can see that use this step in same scenario as well: https://github.com/studentquiz/moodle-mod_studentquiz/blob/bca9dc6b33e6128edaf84d8f8055217719ccfe30/tests/behat/commen_history_backup_restore.feature#L20
Unfortunately, we don't really have shorter step in the boost theme to use.
Because the step above need to be in the restore page, as the doc said: "You should be in the 'Restore' page where the backup is." and to go to the restore page in boost theme, we need those two step above.
Is it ok that we keep those steps?

Reply:
Hi Tim, just ignore my comment.
I found the steps to do this thank for your sharing on the behat improvement on the SIG meeting.
I'm update the behat test now.